app/src/ui/lib/dark-theme.ts

```diff
@@ -4,7 +4,10 @@ import {
   isWindows10And1809Preview17666OrLater,
 } from '../../lib/get-os'
 
-export function supportsDarkMode() {
+/**
+ * Whether or not the current OS supports System Theme Changes
+ */
+export function supportsSystemThemeChanges() {
```

Comment:
Something of note, this doesn’t have to be here. From what Electron shows we can use system theme in older OS versions but of course setting it as such is effectively the same as Light mode. I think for that reason it might be worth keeping this around. I sorta suspect that Chrome Dev tools have System Theme as a possible setting on older OSes despite the fact that the value would never change. I’m not sure what the older OS would do as I don’t have a machine on an older OS to test this.